Week3/script.js

```diff
@@ -0,0 +1,278 @@
+// JS1 Homework Week 3
+
+// 1. Strings!
+// 1.1 Add the string to your file and log it.
+let myString = 'hello,this,is,a,difficult,to,read,sentence';
+console.log(myString);
+
+// 1.2 Log the length of myString.
+console.log(`The length of myString is: ` + myString.length);
+
+// 1.3 The commas make that the sentence is quite hard to read.
+// Find a way to remove the commas from the string and replace them with spaces.
+// 1.4 Log myString to see if you succeeded.
+console.log(myString.replace(/,/g, ` `));
+// or:
+console.log(myString.split(`,`).join(` `));
+
+console.log('.............................................');
+
+// 2. Arrays!
+let favoriteAnimals = ['blowfish', 'capricorn', 'giraffe'];
+
+// 2.1 Add a statement that adds Mauro's favorite animal 'turtle' to the existing array.
+console.log(favoriteAnimals.push('turtle'));
```

Comment:
You don't have to console.log everything, this statement doesn't need to be logged to work.